MAINTENANCE.md

```diff
@@ -208,3 +209,75 @@ Here are some of the shortcuts included in [`bashrc_for_gmt`](/admin/bashrc_for_
 - `vpngdbuild` and `vpdfdbuild` can be used to open the results from all failing image-based tests.
 - `view_png_failures_r` and `view_pdf_failures_r` can be used for view failures of the release build with a lag between
   opening each file.
+
+## Continuous Integration
+
+We use GitHub Actions Continuous Integration (CI) services to build and test the project on Linux, macOS and Windows.
+
+There are 11 configuration files located in `.github/workflows/`:
+
+1. `backport.yml` (Backports PRs into the 6.1 branch)
+
+  This workflow backports PRs labels by "backport 6.1" into the 6.1 branch.
+
+2. `build.yml` (Build GMT and run a few simple tests)
+
+  This workflow is run on every commit to the *master* branch and Pull Request branches. The workflow configures
+  and builds GMT on Linux, macOS, and Windows and runs some simple tests. The workflow uses the scripts in the `ci/`
+  directory for downloading/installing dependencies (`ci/download-coastlines.sh` and `ci/install-dependencies-*.sh`),
+  configuring GMT (`ci/config-gmt-*.sh`), building GMT (`ci/build-gmt.sh`), and running some simple
+  tests (`ci/simple-gmt-tests.*`).
```

Comment:
The `ci/build-gmt.sh` is not used by the GMT project. It's only used by external projects like PyGMT. 
```suggestion
  This workflow is run on every commit to the *master* branch and Pull Request branches. The workflow configures
  and builds GMT on Linux, macOS, and Windows and runs some simple tests. The workflow uses the scripts in the `ci/`
  directory for downloading/installing dependencies (`ci/download-coastlines.sh` and `ci/install-dependencies-*.sh`),
  configuring GMT (`ci/config-gmt-*.sh`), building GMT, and running some simple
  tests (`ci/simple-gmt-tests.*`).
```